Use kubectl scale (#932)
SUSE/doc-caasp: adoc/deployment-sysreqs.adoc

The following describes two methods for replica management if you wish to work with a cluster below the default replica size requirement.

> ===== Grow or Shrink replica number

Reviewer: ```suggestion
===== Update replica number
```

> One method is to grow or shrink the number of overall replicas being created by a service.

Reviewer: ```suggestion
One method is to update the number of overall replicas being created by a service.
```

Please consult the documentation of your respective service what the replica limits for proper high availability are.
In case the replica number is too high for the cluster, you must increase the cluster size to provide more resources.

> . Grow or shrink deployment replica size before node joining.

Reviewer: ```suggestion
. Update deployment replica size before node joining.
```
